### DISCUSSION TITLE
제페토 아바타 애니메이션 관련 질문드립니다.

### DISCUSSION THREAD

**Discussion by yeowu** [naverz/zepeto-studio-kor#12] [5. Q&A]
제페토 아바타의 모션을 유니티 애니메이터로 편집/제작하는 방법을 알고싶습니다.
그외 모션 관련해서 참고할만한 자료나 튜토리얼이 있다면 공유해주시면 감사하겠습니다.

**Comment by x4cf** [comment]
모션은 블렌더나 마야, 3D멕스에서 제작한다음 유니티에서 사용하는편이 좋습니다.

https://studio.zepeto.me/guides/creating-your-item-3d-modelers-1
에 들어가셔서
creatorBaseSet_zepeto.zip 파일 (다운로드) - 2021.08 updated 파일을 다운로드 받으신다음 작업하시면됩니다
